test/integration_test/basic_test.go

```diff
@@ -100,6 +98,20 @@ var testStorageClusterBasicCases = []types.TestCase{
 					Enabled: true,
 				},
 			}
+			trueVal := true
+			cluster.Spec.Storage = &corev1.StorageSpec{
+				UseAll:               nil,
+				UseAllWithPartitions: &trueVal,
+				ForceUseDisks:        &trueVal,
+				Devices:              nil,
+				CacheDevices:         nil,
+				JournalDevice:        nil,
+				SystemMdDevice:       nil,
+				KvdbDevice:           nil,
+			}
+			cluster.Spec.DeleteStrategy = &corev1.StorageClusterDeleteStrategy{
+				Type: corev1.UninstallAndWipeStorageClusterStrategyType,
+			}
```

Comment:
The UninstallStorageCluster function will automatically add the UninstallAndWipe delete strategy. We don't need to add it here.

Reply:
It is useful to run on dev env. without wiping after some runs it would fail due to no more available drives

Reply:
You need to get a cluster with backing drives.
Also, if the wipe fails on your cluster so often and if it is a valid issue, we should be filing a PWX ticket.

Reply:
the wipe did not fail